### DISCUSSION TITLE
Working version with HmIP-VDMOT

### DISCUSSION THREAD

**Discussion by dragop4** [nliaudat/esp32_8ch_motor_shield#31] [General]
Hi,

Which version from EasyEDA is well working with HmIP-VDMOT?

I have v54 rev 1.35 and i have problem with it.

**Comment by nliaudat** [comment]
The AC version PCB v55 rev 1.45 was tested 4 days ago https://github.com/nliaudat/esp32_8ch_motor_shield/discussions/19#discussioncomment-4412833
